cmd/block_read_poc/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// NewLogger ... new logger initialization
+func NewLogger() (*zap.Logger, error) {
+	var config zap.Config
+	var options []zap.Option
+
+	config = zap.NewDevelopmentConfig()
+	config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+	options = append(options, zap.AddStacktrace(zap.FatalLevel))
+
+	return config.Build(options...)
+}
```

Comment:
Let's move this out of the `cmd` directory and reroute within `/internal`.

Reply:
Or even a public `/pkg` repo routing that way other developers could import our logger wrapper if needed

Reply:
@epociask which would you prefer. I will move to internal for now.